tests/test_views.py

```diff
@@ -1281,3 +1316,193 @@ def test_get_course_specific_consent_for_enrolled_user(
             'http://localhost:8000/courses/{course_id}/courseware'.format(course_id=course_id),
             fetch_redirect_response=False,
         )
+
+    @mock.patch('enterprise.views.render', side_effect=fake_render)
+    @mock.patch('enterprise.views.configuration_helpers')
+    @mock.patch('enterprise.tpa_pipeline.get_enterprise_customer_for_request')
+    @mock.patch('enterprise.views.get_real_social_auth_object')
+    @mock.patch('enterprise.views.quarantine_session')
+    @mock.patch('enterprise.views.lift_quarantine')
+    @mock.patch('enterprise.views.CourseApiClient')
+    @mock.patch('enterprise.views.organizations_helpers')
+    @mock.patch('enterprise.views.CourseCatalogApiClient')
+    def test_get_course_enrollment_page_with_no_course_effort(
+            self,
+            course_catalog_client_mock,
+            organizations_helpers_mock,
+            course_api_client_mock,
+            lift_quarantine_mock,   # pylint: disable=unused-argument
+            quarantine_session_mock,    # pylint: disable=unused-argument
+            social_auth_object_mock,   # pylint: disable=unused-argument
+            get_ec_for_request_mock,   # pylint: disable=unused-argument
+            configuration_helpers_mock,
+            render_to_response_mock,    # pylint: disable=unused-argument
```

Comment:
@replaceafill Instead of adding `# pylint: disable=unused-argument` to all of these, you can add `_unused` as a suffix to the parameter names, see the definition of [`dummy-variables-rgx`](https://github.com/edx/edx-enterprise/blob/master/pylintrc#L155) for this repo.